repos/fdbt-site/src/pages/pointToPointPeriodProduct.tsx

```diff
@@ -0,0 +1,137 @@
+import React, { ReactElement } from 'react';
+import upperFirst from 'lodash/upperFirst';
+import ExpirySelector from '../components/ExpirySelector';
+import TwoThirdsLayout from '../layout/Layout';
+import { POINT_TO_POINT_PRODUCT_ATTRIBUTE, OPERATOR_ATTRIBUTE } from '../constants/attributes';
+import { ErrorInfo, NextPageContextWithSession, PointToPointPeriodProduct } from '../interfaces';
+import CsrfForm from '../components/CsrfForm';
+import FormElementWrapper, { FormErrorBlock, FormGroupWrapper } from '../components/FormElementWrapper';
+import ErrorSummary from '../components/ErrorSummary';
+import { getSessionAttribute } from '../utils/sessions';
+import { getCsrfToken } from '../utils';
+
+const title = 'Point to Point Period Product - Create Fares Data Service';
+const description = 'Point to point period product details entry page of the Create Fares Data Service';
+
+interface PointToPointPeriodProductProps {
+    product: PointToPointPeriodProduct | null;
+    operator: string;
+    passengerType: string;
+    errors: ErrorInfo[];
+    csrfToken: string;
+}
+
+const ProductDetails = ({
+    product,
+    operator,
+    passengerType,
+    csrfToken,
+    errors,
+}: PointToPointPeriodProductProps): ReactElement => {
+    const { productName } = product || {};
+    const periodValue = product?.productDuration || '';
+    const periodUnits = product?.productDurationUnits || undefined;
```

Comment:
You shouldn't need || if you've done optional chaining

Reply:
For the period value, won't it error if we let it default to undefined instead of an empty string?

Reply:
Yeah the empty string one is probably ok